x/distribution/keeper/keeper_test.go

```diff
@@ -3,36 +3,67 @@ package keeper
 import (
 	"testing"
 
-	"github.com/stretchr/testify/require"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+	"github.com/stretchr/testify/require"
 )
 
-func TestSetGetPreviousProposerConsAddr(t *testing.T) {
-	ctx, _, keeper, _, _ := CreateTestInputDefault(t, false, 0)
+func TestWithdrawDelegationRewards(t *testing.T) {
```

Comment:
Looks like this test still needs to be completed?

Reply:
It got moved to another file (just deleted).